### PR TITLE
resolve permission denied

### LINKED ISSUE #7: Build issue
### Name and Version

-

### What steps will reproduce the bug?

-

### What is the expected behavior?

https://github.com/milvus-io/bitnami-docker-etcd/pull/6

### What do you see instead?

-

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jenkoon**
FIX https://github.com/milvus-io/bitnami-docker-etcd/issues/7

### CODE REVIEW

**Comment by jenkoon** [comment]
@haorenfsa please review, I can build and start etcd server in my local env

**Comment by haorenfsa** [comment]
@weiZhenkun the problem is in file `3.5/debian-12/prebuildfs/usr/sbin/install_packages`, which does not have `x` flag.

File: 3.5/debian-12/Dockerfile [hunk 1]
```diff
@@ -50,6 +51,7 @@ RUN apt-get autoremove --purge -y curl ca-certificates && \
 RUN chmod g+rwX /opt/bitnami
```
Comment (haorenfsa) [inline 3.5/debian-12/Dockerfile:51]: already give `x` here.

**Review by haorenfsa — CHANGES_REQUESTED** [review]

File: 3.5/debian-12/Dockerfile (hunk 1 above)
Comment (haorenfsa) [reply 3.5/debian-12/Dockerfile:51]: The scripts mod better be altered in repo instead.

File: 3.5/debian-12/Dockerfile [hunk 2]
```diff
@@ -50,6 +51,7 @@ RUN apt-get autoremove --purge -y curl ca-certificates && \
...
 
 COPY rootfs /
+RUN chmod +x /opt/bitnami/scripts/etcd/*
```
Comment (haorenfsa) [inline 3.5/debian-12/Dockerfile:54]: The scripts mod better be altered in repo instead.

**Review by haorenfsa — CHANGES_REQUESTED** [review]

**Comment by haorenfsa** [comment]
Thank you very much @weiZhenkun, this patch is combined in #9.

**Pull request closed without merging**